docs/toggles/toggles.rst

```diff
@@ -68,3 +68,8 @@ To access your plugin:
   ``toggleEnabled`` method.
 
 (Note: Plugins were historically called Feature Flags and Toggles)
+
+ .. toctree::
```

Comment:
nit: confirming this is toctree and not doctree?

Reply:
Confirming that `toctree` is the correct syntax. It is used to set the tree structure of documents that fall under this one.